content/preferences-settings/processing.md

```diff
@@ -80,3 +81,10 @@ OpenCL scheduling profile
 : - _default_: the GPU processes the center view pixelpipe; the CPU processes the preview pipe; 
 : - _multiple GPUs_: both pixelpipes are processed in parallel on two different GPUs; 
 : - _very fast GPU_: both pixelpipes are processed sequentially on the GPU. 
+
+tune OpenCL performance
+: Defines how darktable will attempt to tune OpenCL performance for your system. The following options are provided (default _nothing_):
+: - _nothing_: do not attempt to tune OpenCL performance.
+: - _memory size_: on the first run of a pixelpipe, darktable determines how much memory is available and assumes that the same amount will continue to be available for the remainder of your darktable session. This will usually be safe if you are not using any other applications that need a reasonable amount of your GPU's memory. Otherwise you might be better to disable this option, as it could lead to out-of-memory errors.
```

Comment:
I would say
1. "This will usually be safe if you are not starting other applications that need a reasonable amount of your GPU's memory while running darktable"
2. " ... as it could lead to out-of-memory errors resulting in a performance drop because of falling back to cpu code."

Also maybe some additional info like
"You may switch this off at any time and on again to re-tune"